ethereum/core/src/main/java/org/hyperledger/besu/ethereum/core/BlockHeader.java

```diff
@@ -306,6 +316,7 @@ public String toString() {
     sb.append("baseFee=").append(baseFee).append(", ");
     sb.append("mixHashOrPrevRandao=").append(mixHashOrPrevRandao).append(", ");
     sb.append("nonce=").append(nonce);
+    sb.append("withdrawalsRoot=").append(withdrawalsRoot).append(", ");
```

Comment:
nit: I mentioned this on Fabio's PR so better mention on here too...should we only be including if it's not null?

Reply:
Yeah that is cleaner